src/main/java/net/neoforged/neoforge/fluids/FluidStack.java

```diff
@@ -423,6 +424,14 @@ public Component getHoverName() {
         return getFluidType().getDescription(this);
     }
 
+    /**
+     * Returns the amount of this stack as a Component.
+     */
+    public Component getHoverAmount() {
```

Comment:
Its probably worth having a method on `FluidStack` to get the full tooltip including display name to standardize the order and color between that and the amount string. Plus would call `appendHoverText`

Reply:
raised for discussion on discord due to the possibility that such a method should also take certain data components into account.